### DISCUSSION TITLE
End4 dots

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [JaKooLit/Fedora-Hyprland#111] [Q&A]
Could you make it able to use [end4s dotfiles](https://github.com/end-4/dots-hyprland) or convert [the script](https://github.com/end-4/dots-hyprland/blob/main/install.sh) to fedora

**Comment by JaKooLit** [comment]
No plan to do that since its no my configs.

Im just inviting a shit load of errors if I mess up or trying to imitate somebody's configs

you should ask end4 instead

**Reply by pizzav-xyz** [reply]
If ender converted it do I run your script and then his script?
